### PR TITLE
Purchase order aggregate changes

### PR DESCRIPTION
**Pull request by YuriiHaidamaka**
In this PR we have changed all classes that relate to the purchase order aggregate as follows:
-Removed unnecessary whitespaces
-Added package-info.java
-Added Javadocs for all public methods and classes
-Validated null arguments in public static methods

### CODE REVIEW

File: api-java/src/main/java/javaclasses/mealorder/PurchaseOrderSender.java [hunk 1]
```diff
@@ -32,14 +31,13 @@
 public interface PurchaseOrderSender {
 
     /**
-     * Creates the Spreadsheet from the order list of the purchase order.
-     * Sends it to vendor email address.
+     * Sends purchase order to the vendor email address.
      *
      * @param purchaseOrder purchase order to form spreadsheet and send
```
Comment (YegorUdovchenko) [inline api-java/src/main/java/javaclasses/mealorder/PurchaseOrderSender.java]: form spreadsheet and - remove

File: api-java/src/main/java/javaclasses/mealorder/ServiceFactory.java [hunk 2]
```diff
@@ -34,8 +36,8 @@
     // TODO 2/20/2018[yegor.udovchenko]: Replace with implementation
```
Comment (YegorUdovchenko) [inline api-java/src/main/java/javaclasses/mealorder/ServiceFactory.java]: Create issue and make a link in //TODO

File: api-java/src/main/java/javaclasses/mealorder/c/po/PurchaseOrders.java [hunk 3]
```diff
@@ -41,65 +39,76 @@
...
-        // Prevent instantiation of this utility class.
+    /** Prevents instantiation of this utility class. */
+    private PurchaseOrders() {
     }
 
     /**
      * Performs the validation of a purchase order creation process.
-     * Checks each order in the list to match purchase order date
+     *
+     * <p></>Checks each order in the list to match purchase order date
```
Comment (vladislav-kozachenko) [inline api-java/src/main/java/javaclasses/mealorder/c/po/PurchaseOrders.java]: remove </>

File: api-java/src/main/java/javaclasses/mealorder/PurchaseOrderSender.java [hunk 4]
```diff
@@ -32,14 +31,13 @@
 public interface PurchaseOrderSender {
 
     /**
-     * Creates the Spreadsheet from the order list of the purchase order.
-     * Sends it to vendor email address.
+     * Sends purchase order to the vendor email address.
      *
-     * @param purchaseOrder purchase order to form spreadsheet and send
+     * @param purchaseOrder purchase order to form spreadsheet and - remove
```
Comment (vladislav-kozachenko) [inline api-java/src/main/java/javaclasses/mealorder/PurchaseOrderSender.java]: * @param purchaseOrder purchase order to send

**Pull request merged**